src/pages/api/dogs/[id].js

```diff
@@ -0,0 +1,26 @@
+import mongoose from "mongoose";
+import { deleteDog } from "../../../../server/db/actions/Dog";
+
+export default function handler(req, res) {
+  if (req.method == "DELETE") {
+    if (!mongoose.isValidObjectId(req.query.id)) {
+      return res
+        .status(422)
+        .send("Unable to delete because dog id is not in valid format");
```

Comment:
Let's have the api return json in the form:
1. Error occurs
```js
{
  success: false,
  message: "Unable to delete because dog id is not in valid format",
}
```
2. Successful res with data
```js
{
   success: true,
   message: "Dog successfully created",
   data: dogId
}
```